### ISSUE TITLE
Adjust the ad in light of GitHub's new rendering

### ISSUE THREAD

**Issue by jennybc** [tidyverse/reprex#395]
Do I want to adjust the formatting of the ad in light of GitHub's new rendering? By which I mean the square brackets around "reprex package" here:

![Screen Shot 2021-09-27 at 2 20 34 PM](https://user-images.githubusercontent.com/599454/134994761-63424018-53b2-4255-974c-6c2d84950e9e.png)

I thought perhaps it was unintentional but @gaborcsardi points out the css, which suggests it's intentional and here to stay.

```
.markdown-body sup>a::before {
    content: "[";
}
```

Maybe get rid of "the" and "package"? Get rid of the parentheses around the version number?

**Comment by jennybc** [comment]
It seems like GitHub walked back the square brackets thing, which I am glad about.

``` r
x <- 1:5
median(x)
#> [1] 3
x^2
#> [1]  1  4  9 16 25
```

<sup>Created on 2022-08-16 by the [reprex package](https://reprex.tidyverse.org) (v2.0.2)</sup>

**Comment by jennybc** [comment]
I'm leaning towards this:

``` r
x <- 1:5
median(x)
#> [1] 3
```

<sup>Created on 2022-08-16 with [reprex v2.0.1.9000](https://reprex.tidyverse.org)</sup>

**Comment by jennybc** [comment]
To confirm something I had started to suspect/wonder, it is now possible to make hyperlinks in slack messages with markdown-like syntax. Meaning `[text](link)`. This works for people who have opted out of slack's WYSIWYG message formatting, as I have.

https://stackoverflow.com/a/68711870

I'm a little worried that this is not explicitly documented, though:

https://slack.com/help/articles/202288908-Format-your-messages#markup

The SO conversation suggests that it is, which makes me wonder if it was documented and then that was removed, because maybe they aren't committed to supporting that feature? But it does work today and my modification to `advertise` will take advantage of that.
